collectors/ebpf.plugin/ebpf_dcstat.h

```diff
@@ -13,7 +13,6 @@
 #define NETDATA_DC_REQUEST_NOT_FOUND_CHART "dc_not_found"
 
 #define NETDATA_DIRECTORY_CACHE_SUBMENU "directory cache (eBPF)"
```

Comment:
Not sure if we will change this in this PR but since we did it for VFS, do we still want this to have "(eBPF)" in the name?

Reply:
Good point, I will discuss this with @hugovalente-pm in my next meeting with him, but I already added the `(eBPF)` for `VFS`. I changed `dashboard_info` instead the plugin source code, because this simplifies the link creation.